src/screens/LoginScreen/LoginScreen.tsx

```diff
@@ -55,6 +55,7 @@ export default () => {
 				autoCorrect={false}
 				autoFocus={true}
 				blurOnSubmit={false}
+				placeholder = 'example@thebgp.com'
```

Comment:
Make sure to set up eslint auto-formating in your IDE so that these extra spaces around the equal signs are removed automatically. Same for line 71